cmd/aws/lambda/README.md

```diff
@@ -2,13 +2,26 @@
 Contains Substation apps deployed as AWS Lambda functions. All Lambda functions get their configurations from [AWS AppConfig](https://docs.aws.amazon.com/appconfig/latest/userguide/what-is-appconfig.html).
 
 ## autoscaling
-This app handles Kinesis Data Stream autoscaling through SNS notifications and CloudWatch alarms. By default, Kinesis Data Streams will scale up by 50% if a stream exceeds 50% capacity within 1 minute and scale down by 50% if a stream stays below 15% capacity for 1 hour. Capacity is calculated as the maximum of volume (i.e., 60,000 events per minute) or size (i.e., 10GB data per minute).
+This app handles Kinesis Data Stream autoscaling through SNS notifications and CloudWatch alarms. The scaling behavior is to scale up / out if stream utilization is greater than 75% of the Kinesis service limits within a 10 minute period and scale down / in if stream utilization are less than 25% of the Kinesis service limits within a 60 minute period. In both cases, streams scale by 50%.
```

Comment:
The [code](https://github.com/brexhq/substation/pull/27/files#diff-8b56ad3d3680a96061e1e76986e80120f4880b677f428c531e8a9a146bc954f8R17-R18) and [terraform](https://github.com/brexhq/substation/pull/27/files#diff-6cf8c17abe7d725571f53f06bc09d33f95bfb2ec7f02572f30e4b489c330bdd7R19) updates both specify a 2 hour window but you document it as 1 hour here - not sure which you're targeting?

Reply:
It's still TBD based on testing, I'll update the documentation so it's in sync 👍

Reply:
Should be fixed now, it's back to 1 hour.